docs/apps/pennylane.md

```diff
@@ -0,0 +1,79 @@
+---
+tags:
+  - Free
+---
+
+# Pennylane
+
+PennyLane is an open-source software framework for quantum machine learning, 
+quantum chemistry, and quantum computing.
+
+## Available
+
+Currently supported pennylane versions:
+
+| Version | Module                               | LUMI  | Notes           |
+|:--------|:-------------------------------------|:-----:|-----------------|
+| 0.33.0  | `pennylane-lightning-kokkos/0.33.0`  | X     | default version |
```

Comment:
I believe `pennylane-lightning-kokkos>0.32` is deprecated. The kokkos accelerated part of pennylane-lightning is now provided through `pennylane-lightning>=0.33` https://github.com/PennyLaneAI/pennylane-lightning/releases/tag/v0.33.0

Reply:
I think so. Yeah it's confusing. If 0.33 works well then 0.32 and 0.31 could be removed?